cns/restserver/api_test.go

```diff
@@ -1292,6 +1292,37 @@ func TestCreateHostNCApipaEndpoint(t *testing.T) {
 	fmt.Printf("createHostNCApipaEndpoint Responded with %+v\n", createHostNCApipaEndpointResponse)
 }
 
+// TestNCIDCaseInSensitive() tests NCID case insensitive that either uppercase or lowercase NCID should be accepted
+// get the ncVersionList with nc GUID as lower case and when looking up if the ncid is present in ncVersionList,
+// convert it to lowercase and then check if Vfp programming completes
+func TestNCIDCaseInSensitive(t *testing.T) {
+	setEnv(t)
+	err := setOrchestratorType(t, cns.Kubernetes)
+	if err != nil {
+		t.Fatalf("TestNCIDCaseSensitive failed with error:%+v", err)
+	}
+
+	// add a list of NCIDs with upper-case NCIDs, lower-case NCIDs, upper-case cns-managed mode NCID and lower-case cns-managed mode NCID
+	ncids := []string{
+		"Swift_89063DBF-AA31-4BFC-9663-3842A361F189", "Swift_3AD36FBD-BB13-5FAA-7717-2483B451A236", "Swift_f5750a6e-f05a-11ed-a05b-0242ac120003",
+		"Swift_fd578a90-f05a-11ed-a05b-0242ac120003", "17289C8E-F05B-11ED-A05B-0242AC120003", "0f6d764a-f05b-11ed-a05b-0242ac120003",
```

Comment:
we only need one of these - look copy paster error
Swift_f5750a6e-f05a-11ed-a05b-0242ac120003
Swift_fd578a90-f05a-11ed-a05b-0242ac120003